docs/spark/how-to-guides/debug.md

```diff
@@ -24,7 +24,7 @@ spark-submit \
 
 When you run the command, you see the following output:
 
-```
+```console
 ***********************************************************************
 * .NET Backend running debug mode. Press enter to exit *
```

Comment:
@mamccrea unrelated to this PR, but do you know where this message could be updated? back end should be two words.

Reply:
@mairaw, I think it's [here](https://github.com/dotnet/spark/blob/master/src/scala/microsoft-spark-3.0.x/src/main/scala/org/apache/spark/deploy/dotnet/DotnetRunner.scala#L150)

Reply:
Thanks @Youssef1313! I've opened a PR there.